### ISSUE TITLE
Package verifier - log cuts off and test fails well before timout

### ISSUE THREAD

**Issue by TheCakeIsNaOH** [chocolatey/home#26]
Sometimes in the package verifier testing, the install log will cut off at a random point well before the timout limit. So perhaps 10-15 minutes into the test, observed via the timestamps. 

Examples:
[Jellyfin](https://gist.github.com/choco-bot/801544b0d07dfcd17e723eebe6c10e21)
[brim](https://gist.github.com/choco-bot/04d45bd4a88e265d735cb85acc68a9f5)

### **Note:**
If the log ends with a line where the installer for the software is called, it is most likely a problem with the software installer, and not this issue. In most cases where this happens, the installer was not silent, and it is hanging waiting for user input, but in rare cases also possible that it takes so long that it times out the test before it finishes.

**Comment by AdmiringWorm** [comment]
I don't think it is anything to do with the installer itself, but rather something with the VM that the verifier runs on (although I can be wrong about that).

For the brim package, the last log line is from before the script is even called (looks to be the line before the exporting of Chocolatey functions that can be used in a package).

And the jellyfin has log lines to after the dependency was installed (but before doing anything for the next package).

I am guessing that WinRM (or SSH) disconnected between the VM and the host that caused this, so I am guessing the installation continued but since the connection was closed it would still be considered an error.

**Comment by AdmiringWorm** [comment]
I have not seen this happen for a long time now, @TheCakeIsNaOH can you confirm whether this is still an issue or not?

**Comment by pauby** [comment]
@AdmiringWorm This is an ongoing and random issue that happens with the Verifier from time to time. @Windos would be able to shed some more light on it.

There is work on the Verifier image coming up that we _hope_ will fix this problem, however it would be a better use of time to look into it after that works happens.
